### PR TITLE
Add write-up for the Netdata_official rock-on (#253).

### LINKED ISSUE #253: [Rock-on write-up] Netdata (official)
In the pull request updating our Netdata rock-on (https://github.com/rockstor/rockon-registry/pull/237#issuecomment-733132309), @Hooverdan96 brought up the need for further documentation on this rock-on. In particular, it requires the GID for the docker group, which can be retrieved in multiple ways. This should be explained and illustrated in our documentation.

### PR DESCRIPTION
**Pull request by FroggyFlox**
Fixes #253
@phillxnet, ready for review.

### This pull request's proposal
Following the merge of https://github.com/rockstor/rockon-registry/pull/266, we should create a write-up for helping with the installation of the Netdata rock-on. Although the latter is very easy, it does require the user to input the GID for the `docker` group. This write-up is thus a good way to explain that point.

### Sphinx build

Built successfully with Sphinx v3.4.1:

<details><summary>Full Sphinx build log</summary>

```
# make html
sphinx-build -b html -d _build/doctrees   . _build/html
Running Sphinx v3.4.1
making output directory... done
building [mo]: targets for 0 po files that are out of date
building [html]: targets for 64 source files that are out of date
updating environment: [new config] 64 added, 0 changed, 0 removed
reading sources... [100%] windows-shadow-copy                                                                                                                                                                                                
looking for now-outdated files... none found
pickling environment... done
checking consistency... done
preparing documents... done
writing output... [100%] windows-shadow-copy                                                                                                                                                                                                 
generating indices... genindex done
writing additional pages... search done
copying images... [100%] change-password.gif                                                                                                                                                                                                 
copying static files... done
copying extra files... done
dumping search index in English (code: en)... done
dumping object inventory... done
build succeeded.

The HTML pages are in _build/html.

Build finished. The HTML pages are in _build/html.
```

</details>

### Checklist
- [x] With the proposed changes no Sphinx errors or warnings are generated.
- [x] I have added my name to the AUTHORS file, if required (descending alphabetical order).

### CODE REVIEW

File: docker-based-rock-ons/netdata_official.rst [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+.cloud/docs/overview/what-is-netdata>`_.
+
+.. _netdata_doc:
+
+Netdata Documentation
+---------------------
+
+Netdata's features and possibilities can be greatly expanded and customized to
+better fit your needs, hardware, and preferences. Fortunately, Netdata provides
+and `extensive documentation <https://learn.netdata.cloud/>`_ to learn how to do
```
Comment (phillxnet) [inline docker-based-rock-ons/netdata_official.rst:33]:
```suggestion
`extensive documentation <https://learn.netdata.cloud/>`_ to learn how to do
```
Extraneous 'and'.

File: docker-based-rock-ons/netdata_official.rst [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+so. Note that out-of-the-box, Netdata (and thus this rock-on) already gathers a
+very large number of metrics suitable for most users; further customization is
+thus only optional.
```
Comment (phillxnet) [inline docker-based-rock-ons/netdata_official.rst:36]:
```suggestion
thus optional.
```
Just a suggestion as I think it reads better.

**Review by phillxnet — CHANGES_REQUESTED** [review]
@FroggyFlox This is a really welcome addition. Especially given our recent move to the official docker container and the potential incracies introduced with that via the GID requirement. Plus I think our own metrics system is a little wanting/rudimentary for having a well documented Netdata  install is a major plus.

Only some very trivial changes suggested on this one. Nice.

And thanks again for seeing to this significant doc update.

File: docker-based-rock-ons/netdata_official.rst (hunk 1 above)
Comment (FroggyFlox) [reply docker-based-rock-ons/netdata_official.rst:33]: Oops... It was meant to be "an", but it actually works better the way you suggested so I'll commit that.

**Comment by FroggyFlox** [comment]
Thanks again for yet catching all my mistakes, @phillxnet... Sorry i seem to keep making these.

Cheers!

**Comment by phillxnet** [comment]
@FroggyFlox No worries on any mistakes. Particularly tricky to see your own, at least that's what I find.

Thanks again for tending to this doc update. I'm pretty impressed with netdata and it's new cloud feature is very welcome so it good we finally have a write up with links to their docs and the nuance of the GID clarified.

Merging and publishing shortly as discussed.

**Comment by phillxnet** [comment]
@FroggyFlox Again, now published and again apologies for taking so long with reviewing the last 3 of your previously outstanding pull requests.

**Pull request merged**